src/frontend/src/lib/components/registry/teams/RosterList.svelte

```diff
@@ -13,30 +16,33 @@
   <col class="mode" />
   <thead>
     <tr>
-      <th>Tag</th>
-      <th>Name</th>
-      <th>Status</th>
-      <th>Game</th>
-      <th>Mode</th>
+      <th>{$LL.TEAM_LIST.TAG()}</th>
+      <th>{$LL.TEAM_LIST.NAME()}</th>
+      <th>{$LL.TEAM_LIST.STATUS()}</th>
+      <th>{$LL.TEAM_LIST.GAME()}</th>
+      <th>{$LL.TEAM_LIST.MODE()}</th>
     </tr>
   </thead>
   <tbody>
     {#each team.rosters as roster, i}
       <tr class="row-{i % 2}">
         <td>
-          {roster.tag}
+          <Tag {team} />
         </td>
         <td>
           {roster.name}
         </td>
         <td>
-          {roster.is_recruiting ? 'Recruiting' : 'Not Recruiting'}
+          <Badge
+            classId={roster.is_recruiting ? 'recruiting' : 'not_recruiting'}
+            value={roster.is_recruiting ? 'Recruiting' : 'Not Recruiting'}
+          />
         </td>
         <td>
-          {roster.game}
+          <Badge classId={'game_' + roster.game} value={roster.game.toUpperCase()} />
```

Comment:
For an example of what I mentioned earlier, you could instead do <GameBadge game={roster.game}/> here and then the GameBadge component does the figuring out the correct class and value part.